github-overrides.json

```diff
@@ -0,0 +1,4 @@
+{
+  "CloudCV": "Cloud-CV",
```

Comment:
See https://codein.withgoogle.com/api/program/current/organization/ ; this value is already there.